### DISCUSSION TITLE
How to archive repeated tasks and set the next repeat date

### DISCUSSION THREAD

**Discussion by jcyrio** [nvim-orgmode/orgmode#635] [Q&A] [answered]
I have the following in my agenda file:
* TODO test birthday
DEADLINE: <2023-12-11 Mon ++1y>

I open the Org Agenda, move the cursor to the headline, and do org_agenda_archive. The task disappears from the agenda, and also from the agenda file. I would've expected it to stay in the agenda file, and to simply change the date to next year's date.

What am I doing wrong? I'm apologize I couldn't figure it out on my own.

**Comment by kristijanhusak** [comment, marked as answer]
You should mark it as done (`t`). It will automatically increase the date and keep the state in TODO, since it's a repeatable date.

**Reply by rieje** [reply]
How to set TODO state (e.g. mark as DONE) for next repeat within a headline (not in Org Agenda)?

**Comment by jcyrio** [comment]
> You should mark it as done (`t`). It will automatically increase the date and keep the state in TODO, since it's a repeatable date.

Thank you! That worked.
